plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Browse.js

```diff
@@ -81,6 +85,16 @@ Ext.define('NX.coreui.controller.Browse', {
     me.callParent();
 
     me.listen({
+      store: {
+        '#NX.coreui.store.Repository': {
+          load: function() {
+            if (!me.loadedStore) {
+              me.loadedStore = true;
```

Comment:
More cruft. Sorry!

(I was trying to figure out how to load the correct view on load in drilldowns with multiple stores… and failed… this was just the vestigial remnants from that.)